<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(55, 55,55);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(226, 226, 226);</string>
      </property>
      <property name="title">
       <string>Input Files</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListWidget" name="list_widget">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(131, 131, 131);
color: rgb(48, 48, 48);
alternate-background-color: rgb(151, 151, 151);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::WinPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="check_multithreaded">
           <property name="styleSheet">
            <string notr="true">color: rgb(189, 189, 189);</string>
           </property>
           <property name="text">
            <string>MultiThreaded</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="cancel_task">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 85, 0);</string>
             </property>
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="add_file">
             <property name="text">
              <string>Add Files</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="remove_files">
             <property name="styleSheet">
              <string notr="true">color: rgb(217, 217, 217);</string>
             </property>
             <property name="text">
              <string>Remove Selcted</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clear_files">
             <property name="styleSheet">
              <string notr="true">color: rgb(217, 217, 217);</string>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_out">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>333</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(226, 226, 226);</string>
      </property>
      <property name="title">
       <string>Output Files</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>204</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(189, 189, 189);</string>
             </property>
             <property name="text">
              <string>Mode      :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmBox_mode">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(111, 111, 111);
color: rgb(220, 220, 220);</string>
             </property>
             <item>
              <property name="text">
               <string>Exr 2 Tex</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Tex 2 Exr</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(189, 189, 189);</string>
               </property>
               <property name="text">
                <string>Bit Depth:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmBox_bitDepth">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(111, 111, 111);
color: rgb(220, 220, 220);</string>
               </property>
               <item>
                <property name="text">
                 <string>16-bit</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>32-bit</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8-bit</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(217, 217, 217);</string>
             </property>
             <property name="text">
              <string>Output Path (Empty = Source location):</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="output_path_le">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(111, 111, 111);
</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="choos_dir">
               <property name="font">
                <font>
                 <pointsize>7</pointsize>
                </font>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="convert_files">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(217, 217, 217);</string>
               </property>
               <property name="text">
                <string>Convert</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color: rgb(51, 170, 49);</string>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use this tool to convert 'Exr' images to 'Tex' images or vica versa.&lt;/p&gt;&lt;p&gt; it uses Renderman 'ICE' to convert 'Tex' images back to 'Exr' and&lt;/p&gt;&lt;p&gt; renderman's 'txmake' command to convert 'Exr images to 'Tex'.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="add_files">
   <property name="text">
    <string>open</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
